.github/workflows/build-smart-contracts.yml

```diff
@@ -1,39 +1,53 @@
 name: Build smart contracts
 
 on:
-    workflow_call:
-      secrets:
-        HARDHAT_FORKING_URL:
-          required: true
+  workflow_call:
+    secrets:
+      HARDHAT_FORKING_URL:
```

Comment:
We should change this name; it is provider_url (alchemy)

Reply:
Done, renamed from `HARDHAT_FORKING_URL` into `PROVIDER_URL`.